frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -2274,7 +2274,7 @@ function ReaderHighlight:onMoveHighlightIndicator(args)
         local quick_move_distance_dx = self.view.visible_area.w * (1/5) -- quick move distance: fifth of visible_area
         local quick_move_distance_dy = self.view.visible_area.h * (1/5)
         -- single move distance, user adjustable, default value (4) capable to move on word with small font size and narrow line height
-        local move_distance = Size.item.height_default / G_reader_settings:readSetting("highlight_non_touch_factor")
+        local move_distance = Size.item.height_default / G_reader_settings:readSetting("highlight_non_touch_factor", 4)
```

Comment:
Just for info, so you can choose how you want it:
- `G_reader_settings:readSetting("highlight_non_touch_factor", 4)` will save `4` in G_reader_settings - so if you later decide to change the default, past users won't get the new default and will keep using 4
- if you want to avoid that, you can use `(G_reader_settings:readSetting("highlight_non_touch_factor") or 4)` which won't save your `4` in G_reader_settings

Reply:
Nice one, thanks!